### PR TITLE
SRCHX-255 Accessibility fixes for the AIW site.

### PR DESCRIPTION
**Pull request by rehanabbasi**
Changes in this PR implement fixes for the following JIRA tickets.
SRCHX-229
SRCHX-231
SRCHX-232
SRCHX-233
SRCHX-235
SRCHX-236
SRCHX-255

### CODE REVIEW

File: src/app/asset-filters/asset-filters.component.pug [hunk 1]
```diff
@@ -4,7 +4,7 @@
     .filter-group(*ngIf="availableFilters[filterKey] && isArray(availableFilters[filterKey]) && availableFilters[filterKey].length > 0 && !(availableFilters[filterKey].length === 1 && (availableFilters[filterKey][0].name === '' || availableFilters[filterKey][0].count === 0))")
       span.filter-group__label
         h3 {{ 'FILTERS.' + filterKey | translate }}
-          a#buttonCollectionTypeClear.link--accent.small((keydown.enter)="clearAllFilterGroup(filterKey)", [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1", [ngClass]="{ 'disable': !hasAppliedFilters(filterKey) }", (click)="clearAllFilterGroup(filterKey)", attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters") Clear
+          a#buttonCollectionTypeClear.link--accent.small(*ngIf="hasAppliedFilters(filterKey)", (keydown.enter)="clearAllFilterGroup(filterKey)", [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1", (click)="clearAllFilterGroup(filterKey)", attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters") Clear
```
Comment (NazimAli2017) [inline src/app/asset-filters/asset-filters.component.pug]: Can we extract this?

Comment (NazimAli2017) [inline src/app/asset-filters/asset-filters.component.pug]: For accessibility. 
```suggestion
          a#buttonCollectionTypeClear.link--accent.small(*ngIf="hasAppliedFilters(filterKey)", (keydown.enter)="clearAllFilterGroup(filterKey)", [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1", (click)="clearAllFilterGroup(filterKey)", attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters.") Clear
```

File: src/app/asset-filters/asset-filters.component.pug [hunk 2]
```diff
@@ -41,7 +41,7 @@
   .filter-group(*ngIf="availableFilters.dateObj")
     span.filter-group__label
       h3 Date
-        a#buttonDateFilterClear.link--accent.small((keydown.enter)="clearAllFilterGroup('date')", [tabindex]="hasAppliedFilters('date') ? 2 : -1", [ngClass]="{ 'disable': !hasAppliedFilters('date') }", (click)="clearAllFilterGroup('date')", aria-label="Clear, click to clear applied Date filter") Clear
+        a#buttonDateFilterClear.link--accent.small(*ngIf="hasAppliedFilters('date')", (keydown.enter)="clearAllFilterGroup('date')", [tabindex]="hasAppliedFilters('date') ? 2 : -1", (click)="clearAllFilterGroup('date')", aria-label="Clear, click to clear applied Date filter") Clear
```
Comment (NazimAli2017) [inline src/app/asset-filters/asset-filters.component.pug]:
```suggestion
        a#buttonDateFilterClear.link--accent.small(*ngIf="hasAppliedFilters('date')", (keydown.enter)="clearAllFilterGroup('date')", [tabindex]="hasAppliedFilters('date') ? 2 : -1", (click)="clearAllFilterGroup('date')", aria-label="Clear, click to clear applied Date filter.") Clear
```

File: src/app/shared/footer/footer.component.pug [hunk 3]
```diff
@@ -72,8 +72,8 @@
             #google_translate_element.translate-flyout
         .col-md-3.col-6
           div.social-icons
-            a.icon.icon-twitter(target="_blank" href="http://twitter.com/#!/artstor" title="Follow us on Twitter")
-            a.icon.icon-facebook(target="_blank" href="https://www.facebook.com/ARTstor" title="Keep up with Artstor on Facebook")
+            a.icon.icon-twitter(target="_blank" href="http://twitter.com/#!/artstor" aria-label="Follow us on Twitter")
```
Comment (NazimAli2017) [inline src/app/shared/footer/footer.component.pug]:
```suggestion
            a.icon.icon-twitter(target="_blank" href="http://twitter.com/#!/artstor" aria-label="Follow us on Twitter.")
```

File: src/app/shared/footer/footer.component.pug [hunk 4]
```diff
@@ -72,8 +72,8 @@
...
+            a.icon.icon-facebook(target="_blank" href="https://www.facebook.com/ARTstor" aria-label="Keep up with Artstor on Facebook")
```
Comment (NazimAli2017) [inline src/app/shared/footer/footer.component.pug]:
```suggestion
            a.icon.icon-facebook(target="_blank" href="https://www.facebook.com/ARTstor" aria-label="Keep up with Artstor on Facebook.")
```

File: src/assets/i18n/en.json [hunk 5]
```diff
@@ -415,7 +415,7 @@
       "UPLOAD_IMGS": "Upload Images",
       "PC_DESC": "Only you can see this collection. To share these images with others at your institution, add them to a group. Learn more about Personal Collections.&nbsp; <a href=\"http://support.artstor.org/?article-category=09-using-your-own-images\", target=\"_blank\">http://support.artstor.org/?article-category=09-using-your-own-images</a>",
       "HEADING" : "Personal Collection",
-      "DESCRIPTION" : "Only you can see this collection. To share these images with others at your institution, add them to a group.<br/><a href=\"http://support.artstor.org/?article-category=09-using-your-own-images\" target=\"_blank\">Learn more about Personal Collections</a>."
+      "DESCRIPTION" : "Only you can see this collection. To share these images with others at your institution, add them to a group.<br/><a href=\"http://support.artstor.org/?article-category=09-using-your-own-images\" target=\"_blank\"><u>Learn more about Personal Collections</u></a>."
```
Comment (NazimAli2017) [inline src/assets/i18n/en.json]:
```suggestion
      "DESCRIPTION" : "Only you can see this collection. To share these images with others at your institution, add them to a group.<br/><a href=\"https://support.artstor.org/?article-category=09-using-your-own-images\" target=\"_blank\"><u>Learn more about Personal Collections</u></a>."
```

**Review by willschoener — COMMENTED** [review]
Nothing outside of Nazim's comments. Looks good!

File: src/assets/i18n/en.json (hunk 5 above)
Comment (rehanabbasi) [reply src/assets/i18n/en.json]: Changes made in the next commit `4d90ebb`.

File: src/app/asset-filters/asset-filters.component.pug (hunk 1 above)
Comment (rehanabbasi) [reply src/app/asset-filters/asset-filters.component.pug]: Changes made in the next commit `4d90ebb`.

File: src/app/asset-filters/asset-filters.component.pug (hunk 2 above)
Comment (rehanabbasi) [reply src/app/asset-filters/asset-filters.component.pug]: Changes made in the next commit `4d90ebb`.

File: src/app/shared/footer/footer.component.pug (hunk 3 above)
Comment (rehanabbasi) [reply src/app/shared/footer/footer.component.pug]: Changes made in the next commit `4d90ebb`.

File: src/app/shared/footer/footer.component.pug (hunk 4 above)
Comment (rehanabbasi) [reply src/app/shared/footer/footer.component.pug]: Changes made in the next commit `4d90ebb`.

File: src/app/asset-filters/asset-filters.component.pug (hunk 1 above)
Comment (rehanabbasi) [reply src/app/asset-filters/asset-filters.component.pug]: @NazimAli2017 can you elaborate on what exactly you suggest by extracting this?

Comment (NazimAli2017) [reply src/app/asset-filters/asset-filters.component.pug]: Looks like we could just format it instead, the line is pretty long and difficult to read. Something like this:

```pug
          a#buttonCollectionTypeClear.link--accent.small(
            *ngIf="hasAppliedFilters(filterKey)",
            (keydown.enter)="clearAllFilterGroup(filterKey)",
            [tabindex]="hasAppliedFilters(filterKey) ? 2 : -1",
            (click)="clearAllFilterGroup(filterKey)",
            attr.aria-label="Clear, click to clear all selected {{ 'FILTERS.' + filterKey | translate }} filters."
          ) Clear
```

**Review by NazimHAli — APPROVED** [review]
Minor formatting recommendation, looks good!

**Review by willschoener — APPROVED** [review]

File: src/app/asset-filters/asset-filters.component.pug (hunk 1 above)
Comment (rehanabbasi) [reply src/app/asset-filters/asset-filters.component.pug]: Suggested change made in the recent commit `eab197a`.

**Pull request merged**